src/lib/util/time.h

```diff
@@ -42,6 +42,20 @@ RCSIDH(time_h, "$Id$")
 extern "C" {
 #endif
 
+/* Limits of specified-width time types:
+ *
+ *   These #defines specify the minimum and maximum limits
+ *   of each time types declared above.
+ */
+#define FR_TIME_MAX         INT64_MAX
```

Comment:
`fr_time_t` is `uint64_t`, so these should be UINT64 min/max.

Reply:
The _fr_time_t_ is declared as`typedef int64_t fr_time_t;`